packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -263,12 +273,15 @@ class ThemeData {
       disabledColor,
       accentColor,
       accentColorBrightness,
-      indicatorColor,
-      hintColor,
-      hintOpacity,
-      text,
-      primaryTextTheme,
-      primaryIconTheme
+      hashValues( // Too many values.
+        indicatorColor,
+        hintColor,
+        hintOpacity,
+        errorColor,
+        text,
+        primaryTextTheme,
+        primaryIconTheme
+      )
```

Comment:
!! we exceeded the 20 values that hashValues() supports?! Blimey. Filed https://github.com/flutter/flutter/issues/1356.